src/main/java/com/google/sps/data/NSWGovAQDataPoint.java

```diff
@@ -17,6 +17,14 @@ public NSWGovAQDataPoint() {
     this.aqi = -1;
   }
 
+  public int getId() {
```

Comment:
See above comment about setters

Reply:
NSWGovAQDataPoint is a private class because there is no need to make it public / less public code. It also isnt really used in the other methods so it being private doesnt complicate code or make things harder in the methods per say.

Reply:
This class is only used as a data record, so should be treated as a PDS.

No code is even better than private code. By making the data members public, they can be directly access, and no getters (or any other code) are required.

If you want to make the NSWGovAQDataPoint class "private", that would be best accomplished by making it a nested class (also called inner class) of the NSWGovAdapter. (https://docs.oracle.com/javase/tutorial/java/javaOO/nested.html)

Reply:
I actually think the way Rossana is using by making it a separate class is better for clean code, please correct me if i'm wrong, my impression is this PDS will be used between servlets in the Java side, it does not belong to a adaptor class only e.g. NSWGovAdaptor, so it may not suitable to make it a nested class. 

Create Java Bean style classes is also not a bad idea, have setter / getter can improve encapsulation, although in this case, as a PDS, it does not make any difference, but insist a consistent style is encouraged.

Reply:
The NSW Government data is requested by the adapter class, and only used internally. The common format used by the cache and servlets is the AQDataPoint, the adapter converts from this (internal) type, to the common AQDataPoint type.

I wasn't advocating to make this an internal class; I do agree that keeping it separate is entirely fine. My comment was intended to provide another direction to use if the desire was to make the class private for whatever reason. I don't think that's required, but it's not my code or design.

Reply:
Okay, that makes sense to me, thanks for the explanation, Sai